src/supernova/utils.rs

```diff
@@ -0,0 +1,179 @@
+use bellpepper_core::{
```

Comment:
Nit: If this is used only from the supernova circuit, it would make sense to make circuit_supernova.rs a module and then put this file inside that module.